charts/internal/seed-controlplane/charts/csi-driver-controller/templates/deployment-csi-driver-controller.yaml

```diff
@@ -93,15 +90,17 @@ spec:
         - --default-fstype=ext4
         - --leader-election
         - --leader-election-namespace=kube-system
-        - --timeout={{ .Values.timeout }}
-        - --v=5
+        {{- if .Values.csiProvisioner.extraArgs.timeout }}
+        - --timeout={{ .Values.csiProvisioner.extraArgs.timeout }}
+        {{- end }}
+        {{- if .Values.csiProvisioner.extraArgs.verbosity }}
+        - --v={{ .Values.csiProvisioner.extraArgs.verbosity }}
+        {{- end }}
```

Comment:
Instead of this we could also do:
```
            {{- range $key, $value := .Values.csiProvisioner.extraArgs }}
            - "-{{ $key }}={{ $value }}"
            {{- end }}
````

Same for all the other components obviously, would get rid of some further dry code.

Reply:
@dkistner Do you want me to add the loop?

Reply:
Yes, would reduce also some dry code.